sdks/builds/mac.mk

```diff
@@ -0,0 +1,56 @@
+
+mac_PLATFORM_BIN=$(XCODE_DIR)/Toolchains/XcodeDefault.xctoolchain/usr/bin
+
+##
+# Parameters
+#  $(1): target
+define MacTemplate
+
+_mac-$(1)_CC=$$(CCACHE) $$(mac_PLATFORM_BIN)/clang
+_mac-$(1)_CXX=$$(CCACHE) $$(mac_PLATFORM_BIN)/clang++
+
+_mac-$(1)_AC_VARS= \
+	ac_cv_func_fstatat=no \
+	ac_cv_func_readlinkat=no \
+	ac_cv_func_futimens=no \
+	ac_cv_func_utimensat=no
+
+_mac-$(1)_CFLAGS= \
+	-arch $(2)
+
+_mac-$(1)_CXXFLAGS= \
+	-arch $(2)
+
+_mac-$(1)_CPPFLAGS= \
+	-isysroot $(XCODE_DIR)/Platforms/MacOSX.platform/Developer/SDKs/MacOSX$(MACOS_VERSION).sdk \
+	-mmacosx-version-min=$(MACOS_VERSION_MIN) \
+
+_mac-$(1)_LDFLAGS= \
+	-Wl,-no_weak_imports
+
+_mac-$(1)_CONFIGURE_FLAGS= \
+	--disable-boehm \
+	--disable-btls \
+	--disable-iconv \
+	--disable-mcs-build \
+	--disable-nls \
+	--enable-maintainer-mode \
+	--with-glib=embedded \
+	--with-mcs-docs=no
+
+# _ios-$(1)_CONFIGURE_FLAGS += --enable-extension-module=xamarin --enable-extension-module
```

Comment:
@rolfbjarne yep, how it's going to be done is not figured out yet. But I thought getting this through first wouldn't hurt, especially since it's a difference archive than iOS and as long as mac doesn't use it it's not going to impact you in any way.